client/cli/cli.go

```diff
@@ -35,12 +35,33 @@ import (
 
 const (
 	logFileName = "sliver-client.log"
+
+	configFlagStr        = "config"
+	preludeServerFlagStr = "server"
+	rangeFlagStr         = "range"
+	aesKeyFlagStr        = "aes-key"
 )
 
 var (
 	sliverServerVersion = fmt.Sprintf("v%s", version.FullVersion())
 )
 
+func init() {
+
+	// Import
+	rootCmd.AddCommand(cmdImport)
+
+	// Prelude
+	cmdPrelude.Flags().StringP(configFlagStr, "c", "", "sliver client config file path")
+	cmdPrelude.Flags().StringP(preludeServerFlagStr, "s", "", "prelude server connection string")
+	cmdPrelude.Flags().StringP(rangeFlagStr, "r", "", "prelude range")
```

Comment:
I'll need to double check but I think they removed the concept of ranges, so we can safely delete that.